compiler/passes/resolveIntents.cpp

```diff
@@ -275,7 +275,9 @@ void resolveArgIntent(ArgSymbol* arg) {
       // records/unions.
       bool addedTmp = (isRecord(arg->type) || isUnion(arg->type));
       FnSymbol* fn = toFnSymbol(arg->defPoint->parentSymbol);
-      if (fn->hasFlag(FLAG_EXTERN))
+      // TODO: Consider adding flag for export wrappers instead of this?
+      bool isExportAndLibrary = fLibraryCompile && fn->hasFlag(FLAG_EXPORT);
```

Comment:
It seems to me this shouldn't depend on fLibraryCompile.

Reply:
It's true, it really shouldn't. The condition I'm really trying to test for here is "is this thing is an export wrapper", (as export wrappers are only created during `--library`).

Reply:
I can write a patch that adds a "FLAG_EXPORT_WRAPPER", or something of the sort and uses that instead.